token/core/fabtoken/setup.go

```diff
@@ -38,30 +47,41 @@ func NewPublicParamsFromBytes(raw []byte, label string) (*PublicParams, error) {
 	return pp, nil
 }
 
+// Identifier returns the label associated with the PublicParams
+// todo shall we used Identifier instead of Label?
```

Comment:
todo

Reply:
refactoring to be handled in a different PR.